tests/integration/utils2.go

```diff
@@ -58,6 +61,13 @@ const (
 	badgerFileType DatabaseType = "badger-file-system"
 )
 
+type ClientType string
+
+const (
+	goClientType   ClientType = "go"
+	httpClientType ClientType = "http"
```

Comment:
todo: I think the new consts would benefit from a line or two of documentation each

Reply:
Added